javascript/ql/src/semmle/javascript/Expr.qll

```diff
@@ -1263,13 +2114,41 @@ class ComprehensionBlock extends @comprehensionblock, Expr {
   }
 }
 
-/** A `for`-`in` comprehension block. */
+/**
+ * A `for`-`in` comprehension block in a comprehension expression.
+ *
+ * Example:
+ *
+ * ```
+ * for (x in o)
```

Comment:
Could we make the comprehension syntax examples not coincide with an ordinary `for` loop?